packages/hash/frontend/src/theme/components/index.ts

```diff
@@ -0,0 +1,8 @@
+import { ThemeOptions } from "@mui/material";
+
+import { MuiCssBaselineThemeOptions } from "./utils/MuiCssBaselineThemeOptions";
+
+export const components: ThemeOptions["components"] = {
+  /** ===== UTILS ===== */
+  MuiCssBaseline: MuiCssBaselineThemeOptions,
+};
```

Comment:
Nit pick: can we add all the categories now (just so we don't have to copy them over one by one down the line)?
```suggestion
export const components: ThemeOptions["components"] = {
  /** ===== INPUTS ===== */
  /** ===== DATA DISPLAY ===== */
  /** ===== FEEDBACK ===== */
  /** ===== SURFACES ===== */
  /** ===== NAVIGATION ===== */
  /** ===== LAYOUT ===== */
  /** ===== UTILS ===== */
  MuiCssBaseline: MuiCssBaselineThemeOptions,
  /** ===== DATA GRID ===== */
};
```